tests/aeroval/test_coldatatojson_helpers.py

```diff
@@ -91,14 +91,14 @@ def test__add_entry_json(tmp_path: Path):
 
     result = dict(value=None)
     obs, obs_var, vert, model, model_var = "obs", "obs_var", "Column", "model", "model_var"
-    _add_entry_json(path, result, obs, obs_var, vert, model, model_var)
+    _add_hm_entry_json(path, result, obs, obs_var, vert, model, model_var)
```

Comment:
if the function under test changes name, the testing function should also change name...